v0.3/handbook/libraries/stdlib.md

```diff
@@ -1400,3 +1405,55 @@ bind
     </td>
   </tr>
 </table>
+
+## System
+
+The system library provides various system-level utilities for Eve.
+
+<table class="libitem">
+  <tr>
+    <td colspan="2">
+      <b>#system/timer</b> - starts a timer at the specified resolution
+    </td>
+  </tr>
+
+  <tr>
+    <td>
+      <ul>
+        <li><strong>resultion</strong> - the frequency in milliseconds of the timer.</li>
```

Comment:
read attributes are always optional. Rob was including that for write attributes I think. We also need a clearer way of separating read and write attributes, this would probably be very confusing to somebody unfamiliar with how the timer works.